install_config/configuring_openstack.adoc

```diff
@@ -40,7 +39,55 @@ Consult your OpenStack administrators for values of the `*OS_*` variables, which
 are commonly used in OpenStack configuration.
 
 [[openstack-configuring-masters]]
-== Configuring Masters
+== Configuring OpenStack for {product-title} Masters
```

Comment:
I am not sure this section is needed. We don't have subject or section headers like this on the other docs, where we are providing this content.